.github/workflows/ai-dev-workflow.yml

```
@@ -0,0 +1,164 @@
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '22'
+
+      - name: Setup Antfarm
+        run: |
+          git clone https://github.com/snarktank/antfarm.git /tmp/antfarm
+          cd /tmp/antfarm && npm install && npm run build && npm link
+
+      - name: Run Security Audit
+        run: |
```

actionlint (1.7.10)

[error] "github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)